pgx-macros/src/lib.rs

```diff
@@ -958,10 +970,19 @@ enum DogNames {
 ```
 
 */
-#[proc_macro_derive(PostgresHash)]
+#[proc_macro_derive(PostgresHash, attributes(pgx))]
 pub fn postgres_hash(input: TokenStream) -> TokenStream {
     let ast = parse_macro_input!(input as syn::DeriveInput);
-    impl_postgres_hash(ast).into()
+    let to_sql_config = match sql_entity_graph::ToSqlConfig::from_attributes(ast.attrs.as_slice()) {
```

Comment:
Do you think we could move this into the `impl_postgres_hash` function? I think it'd be more testable. (I do aspire to get these well tested one day...)